src/rpcblockchain.cpp

```diff
@@ -757,6 +761,16 @@ UniValue getchaintips(const UniValue& params, bool fHelp)
     return res;
 }
 
+UniValue mempoolInfoToJSON()
+{
+    UniValue ret(UniValue::VOBJ);
+    ret.push_back(Pair("size", (int64_t) mempool.size()));
```

Comment:
Eh? Please read the whole diff (I did not change that code at all).

Space before the cast? I hope we do not have a rule do not add a space after comma...